### ISSUE TITLE
Problem installing

### ISSUE THREAD

**Issue by Tarang** [spmason/qif2json#3]
There are errors while installing on Node v0.12.0

```
623 warnings generated.
  CXX(target) Release/obj.target/iconv/binding.o
../binding.cc:19:21: error: expected class name
class Iconv: public ObjectWrap {
                    ^
../binding.cc:22:34: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> New(const Arguments& args);
                                 ^~~~~~~~~
                                 v8::internal::Arguments
/Users/user/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../binding.cc:23:38: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
  static Handle<Value> Convert(const Arguments& args);
                                     ^~~~~~~~~
                                     v8::internal::Arguments
/Users/user/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../binding.cc:135:7: error: no member named 'AdjustAmountOfExternalAllocatedMemory' in 'v8::V8'
  V8::AdjustAmountOfExternalAllocatedMemory(-(sizeof(Buffer) + (size_t) hint));
  ~~~~^
../binding.cc:135:54: error: unexpected namespace name 'Buffer': expected expression
  V8::AdjustAmountOfExternalAllocatedMemory(-(sizeof(Buffer) + (size_t) hint));
                                                     ^
../binding.cc:147:9: error: no member named 'AdjustAmountOfExternalAllocatedMemory' in 'v8::V8'
    V8::AdjustAmountOfExternalAllocatedMemory(sizeof(Buffer) + outlen);
    ~~~~^
../binding.cc:147:54: error: unexpected namespace name 'Buffer': expected expression
    V8::AdjustAmountOfExternalAllocatedMemory(sizeof(Buffer) + outlen);
                                                     ^
../binding.cc:148:70: error: no member named 'handle_' in 'v8::Object'
    return Buffer::New(output, outlen, FreeMemory, (void *) outlen)->handle_;
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  ^
../binding.cc:151:12: error: use of undeclared identifier 'ThrowException'; did you mean 'ErrnoException'?
    return ThrowException(ErrnoException(EINVAL, "iconv", "Incomplete character sequence."));
           ^~~~~~~~~~~~~~
           ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: 'ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:151:12: error: too few arguments to function call, expected at least 2, have 1; did you mean '::ErrnoException'?
    return ThrowException(ErrnoException(EINVAL, "iconv", "Incomplete character sequence."));
           ^~~~~~~~~~~~~~
           ::ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: '::ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:154:12: error: use of undeclared identifier 'ThrowException'; did you mean 'ErrnoException'?
    return ThrowException(ErrnoException(errno, "iconv", "Illegal character sequence."));
           ^~~~~~~~~~~~~~
           ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: 'ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:154:12: error: too few arguments to function call, expected at least 2, have 1; did you mean '::ErrnoException'?
    return ThrowException(ErrnoException(errno, "iconv", "Illegal character sequence."));
           ^~~~~~~~~~~~~~
           ::ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: '::ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:157:9: error: no member named 'LowMemoryNotification' in 'v8::V8'
    V8::LowMemoryNotification();
    ~~~~^
../binding.cc:158:12: error: use of undeclared identifier 'ThrowException'; did you mean 'ErrnoException'?
    return ThrowException(ErrnoException(errno, "iconv", "Out of memory."));
           ^~~~~~~~~~~~~~
           ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: 'ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:158:12: error: too few arguments to function call, expected at least 2, have 1; did you mean '::ErrnoException'?
    return ThrowException(ErrnoException(errno, "iconv", "Out of memory."));
           ^~~~~~~~~~~~~~
           ::ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: '::ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:161:12: error: use of undeclared identifier 'ThrowException'; did you mean 'ErrnoException'?
    return ThrowException(ErrnoException(errno, "iconv"));
           ^~~~~~~~~~~~~~
           ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: 'ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:161:12: error: too few arguments to function call, expected at least 2, have 1; did you mean '::ErrnoException'?
    return ThrowException(ErrnoException(errno, "iconv"));
           ^~~~~~~~~~~~~~
           ::ErrnoException
/Users/user/.node-gyp/0.12.0/src/node.h:73:34: note: '::ErrnoException' declared here
NODE_EXTERN v8::Local<v8::Value> ErrnoException(v8::Isolate* isolate,
                                 ^
../binding.cc:165:36: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
Handle<Value> Iconv::Convert(const Arguments& args) {
                                   ^~~~~~~~~
                                   v8::internal::Arguments
/Users/user/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared here
class Arguments;
      ^
../binding.cc:166:15: error: calling a protected constructor of class 'v8::HandleScope'
  HandleScope scope;
              ^
/Users/user/.node-gyp/0.12.0/deps/v8/include/v8.h:816:13: note: declared protected here
  V8_INLINE HandleScope() {}
            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [Release/obj.target/iconv/binding.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1067:12)
gyp ERR! System Darwin 14.1.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/user/Desktop/app/node_modules/qif2json/node_modules/iconv
gyp ERR! node -v v0.12.0
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok 
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "qif2json" "--save"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! iconv@1.2.4 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the iconv@1.2.4 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the iconv package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls iconv
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/user/Desktop/app/npm-debug.log
```

**Comment by spmason** [comment]
Looks like it's iconv that's the problem, you could try a newer version &
open a pr for the package.json bump..

I'll probably have time to try later next week.

**Comment by spmason** [comment]
Published `0.1.1` with all dependencies upgraded to latest (and tests passing in node 0.12 and iojs 1.6)
